src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        return userService.findByEmail(email) != null
+                && userService.findByEmail(email).getPassword().equals(password);
```

Comment:
let's create a variable for the result of findByEmail, so we won't need to call it twice